index.js

```diff
@@ -314,7 +332,9 @@ function whitelistProps (obs) {
   return newObs
 }
 
-function noop () {}
+function noop (v) {
```

Comment:
Nitpick: returning a value is an op! What about having `noop` and `identity` separately?

Reply:
I've often wondered what to call this! Good nit pick 👍